packages/@coorpacademy-components/src/organism/review-slide/index.js

```diff
@@ -37,6 +37,20 @@ const CorrectionPopin = ({
     resultLabel: correctionPopinProps.resultLabel
   };
 
+  // there is an error of eslint here because this useEffect will be only there when the correctionPopin will be there
+  // but that is on purpose to go to the next slide if you press enter when the correctionPopin is present
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
tu peux definir le `useEffect` avant le return, c'est pas grave et tu rentres dans le cadre de `react-hooks/rules-of-hooks`

Reply:
--> vu avec @scalabw effectivement ce n'est pas seulement un effet visuel, il y a un `addEventListener` qu'on ne veut pas si il n'y a pas la popin